notebooks/1_BayesianWorkflow/1_Ins_BayesRefresher.ipynb

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
<p>missing closing parens in the first equation. </p><p><br></p><p>also, i have more than one photo of Pete, who is a lazy puppy.</p>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/bayesian-model-evaluation/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>